src/Sentry/Platforms/Android/Extensions/SentryEventExtensions.cs

```diff
@@ -21,7 +21,8 @@ public static SentryEvent ToSentryEvent(this JavaSdk.SentryEvent sentryEvent, Ja
         using var streamWriter = new JavaOutputStreamWriter(stream);
         using var jsonWriter = new JavaSdk.JsonObjectWriter(streamWriter, javaOptions.MaxDepth);
         sentryEvent.Serialize(jsonWriter, javaOptions.Logger);
-        jsonWriter.Flush();
+
+        streamWriter.Flush();
```

Comment:
`jsonWriter.Flush();` doesn't exist anymore? I'm not sure if we ca skip flushing this and only flush the `streamWriter`

Reply:
Yes, this should be safe as `JsonObjectWriter` and the underlying vendored JSON itself don't hold flush-able state. `sentry-java` in some places flushes the writer` as well (e.g. [see this one](https://github.com/getsentry/sentry-java/blob/87b3774676dde4ad1f5e031bc461e26b9921ad77/sentry/src/main/java/io/sentry/JsonSerializer.java#L222-L225))
But I kinda agree, this feels brittle and error-prone - let me re-add it in the upcoming `7.0.0` release of `sentry-java`. Ideally `JsonObjectWriter` should own the stream and take care of flushing itself.